src/index.css

```diff
@@ -1,3 +1,275 @@
-body {
+* {
```

Comment:
- [ ] [OPTIONAL] GOOD JOB 🎊🥳🎉. One thing actually the space around the app is not necessary. Please kindly remove it ✅

Reply:
Thank you very much!